### ISSUE TITLE
isIn validation example does not work

### ISSUE THREAD

**Issue by sdemjanenko** [christianalfoni/formsy-react#48]
It seems like there is a parse error in `validations="isIn:['foo', 'bar']"`
```
Formsy does not have the validation rule:  'bar']
```

**Comment by sdemjanenko** [comment]
It would be really cool if validations could be an object. e.g. `validations={ {isIn: ['foo', 'bar']} }`

**Comment by christianalfoni** [comment]
Jup, I agree, this fits well with the new `validationErrors` prop on the form. Will include in next release :-)

**Comment by christianalfoni** [comment]
Hi again,

I started to implement this and realized that though an object would be nice it is difficult to translate some values. Like:

isLength:5:12

And most validators are simple strings. Like if you create an app you would rather create your own validators like "isUser", "isProduct" etc.

But I fixed the bug :-)

I want to keep the API as simple as possible, so I think it should be strings or object. not both. I hope to get around to build Formsy React 2, which probably will use objects instead. Learned a lot from inputs on this project. Some good decisions and some bad were made ;-)
